libcudacxx/include/cuda/std/__cccl/builtin.h

```diff
@@ -111,6 +111,12 @@
     NV_IF_ELSE_TARGET(NV_IS_DEVICE, (__builtin_assume(__VA_ARGS__);), (__assume(__VA_ARGS__);))
 #endif // _CCCL_CHECK_BUILTIN(builtin_assume)
 
+#if _CCCL_CHECK_BUILTIN(builtin_assume) || _CCCL_COMPILER(CLANG) || _CCCL_COMPILER(GCC) || _CCCL_COMPILER(NVHPC)
```

Comment:
I believe that line should read 
```suggestion
#if _CCCL_CHECK_BUILTIN(builtin_prefetch) || _CCCL_COMPILER(GCC) ||
```